_posts/2020-07-27-cpp-build-simplification.md

```diff
@@ -0,0 +1,271 @@
+---
+layout: post
+title: "INSERT TITLE HERE"
```

Comment:
Something like

Arrow C++ Build System Enhancements: Better Configurability and easier Minimal or Zero-Dependency Builds

Reply:
I went with "Making Arrow C++ Builds Simpler, Smaller, and Faster", which seemed appropriately more minimal ;)

Reply:
SGTM